molecule_ec2/cookiecutter/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/destroy.yml

```diff
@@ -18,7 +18,7 @@
             skip_instances: true
 
     - name: Destroy molecule instance(s)
-      ec2:
+      community.aws.ec2_instance:
```

Comment:
Use short name and mention collection at top of the playbook, this mode would be backwards compatible, as opposed to the current one.